src/bp/nlu-core/nlu-core.d.ts

```diff
@@ -0,0 +1,210 @@
+/** #######################
```

Comment:
**pointing the attention of reviewers on this file**

(those typings where taken in `botpress.d.ts`)